apis_core/apis_entities/models.py

```diff
@@ -199,22 +198,10 @@ def get_serialization(self):
 
 
 @receiver(post_save, dispatch_uid="create_default_uri")
-def create_default_uri(sender, instance, raw, **kwargs):
-    # with django reversion, browsing deleted entries in the admin interface
-    # leads to firing the `post_save` signal
-    # (https://github.com/etianen/django-reversion/issues/936) - a workaround
-    # is to check for the raw argument
-    if not raw:
-        from apis_core.apis_metainfo.models import Uri
-
-        if kwargs["created"] and sender in caching.get_all_ontology_classes():
-            if BASE_URI.endswith("/"):
-                base1 = BASE_URI[:-1]
-            else:
-                base1 = BASE_URI
-            uri_c = "{}{}".format(
-                base1,
-                reverse("GetEntityGenericRoot", kwargs={"pk": instance.pk}),
-            )
-            uri2 = Uri(uri=uri_c, domain="apis default", root_object=instance)
-            uri2.save()
+def create_default_uri(sender, instance, created, raw, using, update_fields, **kwargs):
+    if getattr(settings, "CREATE_DEFAULT_URI", True):
+        if isinstance(instance, AbstractEntity) and created:
+            base = BASE_URI.strip("/")
+            route = reverse("GetEntityGenericRoot", kwargs={"pk": instance.pk})
```

Comment:
Given that `GetEntityGenericRoot` needs to be implemented by the user, but another route within `apis_core` `GetEntiteGeneric` is automatically there. I suggest the following change:

```python
            try:
                route = reverse("GetEntityGenericRoot",
                                kwargs={"pk": instance.pk})
            except NoReverseMatch:
                route = reverse("apis_core:GetEntityGeneric",
                                kwargs={"pk": instance.pk})
```